### ISSUE TITLE
Pipeline failed: net.sf.saxon.trans.XPathException: Namespace prefix 'xsd' has not been declared

### ISSUE THREAD

**Issue by xquery** [ndw/xmlcalabash1-metadata-extractor#1]
attempting to use this step results in an odd error

ERROR: Pipeline failed: net.sf.saxon.trans.XPathException: Namespace prefix 'xsd' has not been declared
ERROR: Underlying exception: Namespace prefix 'xsd' has not been declared

even when I declare xsd ns, as below, still get the same problem

```
<p:declare-step version="1.0" name="example"
            xmlns:p="http://www.w3.org/ns/xproc"
            xmlns:c="http://www.w3.org/ns/xproc-step"
            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
            xmlns:cx="http://xmlcalabash.com/ns/extensions">
  <p:output port="result" sequence="true"/>  
  <p:import href="library.xpl"/>
  <cx:metadata-extractor href="file:///Users/jfuller/Pictures/scan.jpeg"/>
</p:declare-step>
```

**Comment by ndw** [comment]
My guess is that the problem is in library.xpl...

**Comment by xquery** [comment]
I should have mentioned I added it there and in test xpl invoking cx:metadata-extractor

**Comment by ndw** [comment]
Are you sure you declared it in library.xpl?

It's weird that you don't get the base URI of the document with the error.

**Comment by ndw** [comment]
Ah. The library.xpl file in metadata-extractor-x.x.x.jar is busted.

I'll fix that. I should have tests for each extension too...

**Comment by MikeOcc** [comment]
Is this still an open issue?  I'm getting this error also

net.sf.saxon.trans.XPathException: Prefix ns1 has not been declared

an can't find a way to declare it in code.

**Comment by ndw** [comment]
Oh. Ack. Perhaps I never did fix it. On my list for this weekend.

**Comment by ndw** [comment]
I can't reproduce this with the 1.1.3 version of the extension. @MikeOcc can you send me a small pipeline that reproduces the problem, if you're seeing it with 1.1.3?

**Comment by ndw** [comment]
I'm guessing this is resolved by 1.1.3. If I'm wrong, please reopen it.
